system_tests/common_test.go

```diff
@@ -60,6 +59,172 @@ import (
 type info = *BlockchainTestInfo
 type client = arbutil.L1Interface
 
+type SecondNodeParams struct {
+	nodeConfig           *arbnode.Config
+	execConfig           *gethexec.Config
+	stackConfig          *node.Config
+	dasConfig            *das.DataAvailabilityConfig
+	initData             *statetransfer.ArbosInitializationInfo
+	useBuilderNodeConfig bool
+}
+
+type TestClient struct {
+	ctx      context.Context
+	Client   *ethclient.Client
+	Backend  *eth.Ethereum
```

Comment:
this is only valid for L1, but there is such a thing Backend for the L2 (which can be accessed via execnode).

Let's rename this to L1Backend